DependencyInjection/Compiler/DescriptionEnhancerPass.php

```diff
@@ -26,16 +26,26 @@ public function process(ContainerBuilder $container)
 
         $taggedIds = $container->findTaggedServiceIds('cmf_resource.description.enhancer');
         $enabledEnhancers = $container->getParameter('cmf_resource.description.enabled_enhancers');
+        $validAttributes = ['name', 'alias', 'priority'];
```

Comment:
i find it more readable with the optionsresolver, less likely that we confuse something in the logic. but no big deal, i merged now and we can also leave as is